lib/activerecord-multi-tenant/query_rewriter.rb

```diff
@@ -75,7 +75,7 @@ def visit_Arel_Nodes_Equality(o, *args)
       if o.left.is_a?(Arel::Attributes::Attribute)
         table_name = o.left.relation.table_name
         model = MultiTenant.multi_tenant_model_for_table(table_name)
-        @current_context.visited_handled_relation(o.left.relation) if model.present? && o.left.name == model.partition_key
+        @current_context.visited_handled_relation(o.left.relation) if model.present? && o.left.name.to_s == model.partition_key
```

Comment:
Do we maybe also need `model.partition_key.to_s` - or is that always a string?

Reply:
That's a good idea